src/main/src/test/java/org/geoserver/test/GeoServerAbstractTestSupport.java

```diff
@@ -249,7 +249,7 @@ protected String[] getSpringContextLocations() {
      * "/DEFAULT_LOGGING.properties".
      */
     protected String getLogConfiguration() {
-        return "/TEST_LOGGING.properties";
+        return "TEST_LOGGING.properties";
```

Comment:
Oh so the tests are still using the property files? But in GeoServerSystemTestSupport names without extensions are used instead (so I'm guessing XML files are actually going to be used).

Reply:
We could confirm with debugger; or just change this to ``TEST_LOGGING``.